susemanager-sync-data/additional_products.json

```diff
@@ -1,4 +1,191 @@
 [
+    {
+        "id" : -33,
+        "name" : "Ubuntu 22.04",
+        "identifier" : "Ubuntu-Client",
+        "former_identifier" : "",
+        "version" : "22.04",
+        "release_type" : null,
+        "arch" : "amd64",
+        "friendly_name" : "Ubuntu 22.04",
+        "product_class" : "SLE-M-T-BETA",
```

Comment:
Keeping in mind it won't be beta for 4.3.2, do we want to mark it as beta now :-?

Reply:
Yes, the time until 4.3.2 is too long to make it released. Setting things to released has consequences. Especially that we cannot change all the values if something is found during testing.